server/src/main/java/com/google/speech/tools/voxetta/data/Prompt.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.speech.tools.voxetta.data;
+
+/**
+ * Client-facing schema for Prompts.
+ */
+public class Prompt {
```

Comment:
This is a java object for holding data in the servlet? Or does this also get persisted directly in datastore?

Reply:
rn this pretty much just standardizes the data for GSON when it needs to get sent down, as far as I know the Datastore saving workflow doesn't need this class. this was the recommended practice for the portfolio, so I did it here